Engine/Engine.cs

```diff
@@ -154,7 +154,7 @@ private static RunStepResult BuildSolutionSnapshot(IRunExecutorHost host, RunSta
                     Environment.GetEnvironmentVariable("ProgramFiles(x86)"),
                     string.Format(@"MSBuild\{0}\Bin\msbuild.exe", host.HostVersion)),
                 string.Format(
-                    @"/m /v:minimal /p:Configuration=Debug /p:CreateVsixContainer=false /p:DeployExtension=false /p:CopyVsixExtensionFiles=false /p:OutDir={1} {2}",
+                    @"/m /v:minimal /p:Configuration=Debug /p:CreateVsixContainer=false /p:DeployExtension=false /p:CopyVsixExtensionFiles=false /p:OutDir=""{1}\\"" ""{2}""",

```

Comment:
would this work? i assumed the change would be /p:OutDir=""{1}"" ""{2}""" - i.e. without the '\\'.